Source/Blazorise/Components/Accordion/AccordionToggle.razor.cs

```diff
@@ -41,6 +43,11 @@ protected async Task ClickHandler( MouseEventArgs eventArgs )
             await ParentCollapse.Toggle();
         }
 
+        if ( ParentAccordionItem is not null )
```

Comment:
If might be a little more correct to do an else if. Although it's one of those conditions where you already expect only one of them is to be executed.